tests/sles4sap/saptune/mr_test.pm

```diff
@@ -37,7 +38,10 @@ sub setup {
     quit_packagekit;
     # saptune is not installed by default on SLES4SAP 12 on ppc64le and in textmode profile
     zypper_call "-n in saptune" if ((is_ppc64le() and is_sle('<15')) or check_var('DESKTOP', 'textmode'));
-    zypper_call "in sapconf" if is_sle("<15");
+    if (systemctl("-q is-active sapconf.service", ignore_failure => 1)) {
```

Comment:
Fine but it could be just installed, no need to check anything.

Reply:
> Fine but it could be just installed, no need to check anything.

The whole `if` block will be dropped when the bug is fixed. That's why we need to check.

Reply:
Sure